.github/workflows/release-process.yml

```diff
@@ -37,17 +37,22 @@ jobs:
           ./hack/build-release.sh
           cd cli
           ./hack/build-binaries.sh
-          shasum -a 256 kctrl-* | tee -a ../tmp/checksums.txt
+          shasum -a 256 ./kctrl-* | tee -a ../tmp/checksums.txt
+          echo "# :open_file_folder: Files Checksum" | tee ../tmp/checksums-formatted.txt
+          echo \`\`\` | tee -a ../tmp/checksums-formatted.txt
+          cat ../tmp/checksums.txt | tee -a ../tmp/checksums-formatted.txt
+          echo \`\`\` | tee -a ../tmp/checksums-formatted.txt
 
       - name: Create release draft and upload release yaml
         uses: softprops/action-gh-release@17cd0d34deddf848fc0e7d9be5202c148c270a0a
         with:
           name: ${{ github.ref_name }}
           token: ${{ secrets.GITHUB_TOKEN }}
-          body_path: ./tmp/checksums.txt
+          body_path: ./tmp/checksums-formatted.txt
           files: |
             ./cli/kctrl-*
             ./tmp/release.yml
+            ./tmp/checksums.txt
```

Comment:
whats the rationale for including checksums.txt artifact?

Reply:
It's nice to have for something machine readable/automations. We did once receive a request from a user for `kapp`. All the other repos also have it.